File: Accenture/Que 4/Main.java
import java.util.*;
class Main 
{
    public static int findCount (int arr[], int length, int num, int diff) 
    {
        int count = 0;
        for (int i = 0; i < length; i++)
        {
            if (Math.abs (num - arr[i]) <= diff)
                count++;
        }
        return count>0?count:-1;
    }
    
    public static void main (String[]args) 
    {
        Scanner sc = new Scanner (System.in);
        int n = sc.nextInt ();
        int arr[] = new int[n];
    
        for (int i = 0; i < n; i++)
            arr[i] = sc.nextInt ();
        int num = sc.nextInt ();
        int diff = sc.nextInt ();

        System.out.println (findCount (arr, n, num, diff));
    } 
} 

File: Accenture/Que 4/Main.py
#as same question but in py
def findCount(arr, length, num, diff):
    count = 0 #a var for counting 
    
    for i in range(length):
        absolute_diff = abs(arr[i] - num)  #abs gives no -ve or complex values
        if absolute_diff <= diff:
            count += 1
    
    return count if count > 0 else -1 #if condition fail means

# Test example
arr = [12, 3, 14, 56, 77, 13]
num = 13
diff = 2
result = findCount(arr, len(arr), num, diff)
print("Output:", result)
